StreamTemp-Integ.py

```diff
@@ -98,11 +98,11 @@
         # define training options
         optTrain = default.update(default.optTrainCamels, miniBatch=[BATCH_SIZE, RHO], nEpoch=EPOCH, saveEpoch=saveEPOCH, seed=seed)
         # define output folder for model results
-        exp_name = 'TempDemo'
-        exp_disp = 'FirstRun'
+        exp_name = 'output'
+        #exp_disp = 'demo'
 
 
-        save_path = os.path.join(absRoot, exp_name, exp_disp, \
+        save_path = os.path.join(absRoot, exp_name, \
```

Comment:
how come rootOut doesn't get used in determining the save_path?

Reply:
Hmm, yeah `exp_name` and `rootOut` are redundant.